data/publiccloud/azure_runcmd.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash -u
+set -o pipefail
+. $(dirname "${BASH_SOURCE[0]}")/azure_lib_fn.sh
+# Create a vm and use run-command to execute RunShellscript
+# list, show , delete the created resource
+# set -x
+
+#Required AZ setting variable block
+grpname="${1:-oqaclitest}"
+location="${2:-westus}"
+vmname="${3:-oqacliruncmdvm}"
+ssh_key="${4:-oqaclitest-sshkey}"
+vmximagename="${5:-UbuntuLTS}"
+
+#local variable block
+account="Azure RD"
+admin="azureuser"
+runcmd="oqaclirc"
+
+echo "**Start of AZURE CLI run-command test"
+echo "*************************************"
+
+# Resource creation
+# echo "Set Azure Account { az account set }"
+cmd_status "az_account_set" az account set -n "$account"
+echo "Azure account set to $account"
+
+
+echo "Creating resource group { az group create }"
+cmd_status "az_group_create" az group create -n "$grpname" -l "$location" -o table
```

Comment:
we should take into account probability that execution of previous test may fail in the middle . Which means that clean up logic which is running at the end will not be executed , which means that this line will try to create group with name which already exists so after single glitch all further executions will constantly fail until someone manually will not cleanup all this . 
To workaround this each iteration of script should use random name for all entities created

Reply:
passing resource group from azure_vm_cli which is unique and appended with jobid

Reply:
passing resource group from azure_vm_cli which is unique and appended with jobid